detector3.py

```diff
@@ -93,6 +95,19 @@ def integration_time(self, integration_time):
         
         self._spectrometer.set_integration(self._integration_time*1e6)
 
+    @property
+    def stop_graph(self):
+        return self._stop_graph
+
+
+    @stop_graph.setter
```

Comment:
If we name the property: `stop_graph` all these lines of code are not needed.
Also, generally the property setter is used when we want to add particular function, not just change the value.